eRecruiter.Api/Responses/Job/JobGeoLocationResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace eRecruiter.Api.Responses
+{
+    public class JobGeoLocationResponse
```

Comment:
As we are going to use in other responses `GeoLocationResponse` would be more suitable.

Reply:
done.